src/test/java/org/inferred/freebuilder/processor/AnalyserTest.java

```diff
@@ -460,6 +460,75 @@ public void twoGetters_interface() throws CannotGenerateCodeException {
     assertThat(builder.getGeneratorsByProperty().keySet()).containsExactly(name, age).inOrder();
   }
 
+  @Test
+  public void ignoredEqualsAndHashCode() throws CannotGenerateCodeException {
+    GeneratedBuilder builder = (GeneratedBuilder) analyser.analyse(model.newType(
+        "package com.example;",
+        "public class DataType {",
+        "  @org.inferred.freebuilder.IgnoredByEquals public abstract String getName();",
```

Comment:
`"  @" + IgnoredByEquals.class.getName() + "` works better with automated refactoring and find usage. Also below.